pkg/features/kube_features.go

```diff
@@ -843,6 +843,13 @@ const (
 	//
 	// Enables In-Place Pod Vertical Scaling
 	InPlacePodVerticalScaling featuregate.Feature = "InPlacePodVerticalScaling"
+
+	// owner: @danielvegamyhre
+	// kep: https://kep.k8s.io/4017
+	// beta: v1.28
+	//
+	// Set pod completion index as a pod label for Indexed Jobs and StatefulSets.
+	PodIndexLabel featuregate.Feature = "PodIndexLabel"
```

Comment:
These lists are supposed to be alphabetized.  Can you insert it in the right place?  I know not everything is correct.

Reply:
Sure, done.